src/components/DataTable/Table.js

```diff
@@ -145,8 +145,8 @@ const Headers = ({ headers, advmode, sublists }) => (
 );
 
 const TableRow = ({
-	obj, advmode, headers,
-	rowStyle, rowClassName, category
+	obj, headers,
+	rowStyle, rowClassName, category, advmode
```

Comment:
@Aringoaway Please set it back to its original shape:

```
const TableRow = ({
	obj, advmode, headers,
	rowStyle, rowClassName, category
```

Therefore there are unnecessary changes in the code, since this part can stay unchanged